src/website/views/components/data-table/examples/portal/_second-line-wrapping.pug

```diff
@@ -112,7 +112,7 @@ h3 Second line wrapping
             <div class="chi-data-table__head">
               <div class="chi-data-table__row">
                 <div class="chi-data-table__cell">
-                  <div>Name</div>
+                  <div>This is a long header content with a second line wrapping without tooltip: This is a long content for the tooltip in the wrapped cell</div>
```

Comment:
Include the `-cell-wrap` class in all headers since the prop is set to `true` because without that class it is not possible to wrap headers and cells.

```suggestion
                  <div class="-cell-wrap">This is a long header content with a second line wrapping without tooltip: This is a long content for the tooltip in the wrapped cell</div>
```

Reply:
Done! @josecarlosgonzalezv